client.js

```diff
@@ -0,0 +1,7 @@
+// This work-around needs for working the VSCode's debugging
+try {
+  module.exports = require('./out/src/extension');
+}
+catch {
+  module.exports = require('./dist/node/client/nodeClientMain');
```

Comment:
in general, we need nodeClient only for prod mode, I don't think we need to follow try/catch here, in my understanding we could detect environment to choose actual server file..

Reply:
we resolve server path here: https://github.com/lifeart/vscode-ember/blob/master/src/extension.ts#L35
and this is how "prod" server file looks like: https://github.com/lifeart/vscode-ember/blob/master/start-server.js

Reply:
> in general, we need nodeClient only for prod mode, I don't think we need to follow try/catch here, in my understanding we could detect environment to choose actual server file..

nodeClient needs for debugging extension in VSCode. After running `Launch UELS Extension` VSCode tries open `main` path (`./dist/node/client/nodeClientMain` in 'package.json') but output folder of tsc is `out` and VSCode crashes

Reply:
@kaermorchen I'm pointing to '[start-server.ts](https://github.com/lifeart/ember-language-server/blob/component-context-info-origin/src/start-server.ts)' (compiled version) in linked `lifeart/ember-language-server` repo